### PR TITLE
add provider directory status page for prod

### PR DESCRIPTION
**Pull request by bcastillomartel-va**

### CODE REVIEW

File: pingdom-checks/provider-directory.ping [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
 
+PRODUCTION_PROVIDER_DIRECTORY_API_KEY=$(get-secret "/dvp/production/provider-directory/api-key")
 SANDBOX_PROVIDER_DIRECTORY_API_KEY=$(get-secret "/dvp/lab/provider-directory/api-key")
 PROVIDER_DIRECTORY_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_PROVIDER_DIRECTORY}"
 
+#
+# Production
+#
+
+pingdom save-check \
```
Comment (jhulbert-va) [inline pingdom-checks/provider-directory.ping:11]: I don't think this is available through the gateway:
```
$ curl "https://api.va.gov/services/provider-directory/v0/health" | jq
{
  "message": "no Route matched with those values"
}
```

Comment (bcastillomartel-va) [reply pingdom-checks/provider-directory.ping:11]: what would be the process for adding it?

Comment (jhulbert-va) [reply pingdom-checks/provider-directory.ping:11]: I was informed that Adam is currently tracking that down with Cassowary

**Comment by jhulbert-va** [comment]
Reopen when [this](https://github.com/department-of-veterans-affairs/lighthouse-external-monitoring/pull/93#discussion_r929957444) is resolved.

**Review by jhulbert-va — APPROVED** [review]
I don't love that a healthcheck requires an api-key, but it looks good.

**Comment by jblefkove-va** [comment]
> I don't love that a healthcheck requires an api-key, but it looks good.

I've asked Cassowary to remove the api-key requirement from that endpoint

**Review by golavarr-va — APPROVED** [review]

**Pull request merged**